05_sumAll/sumAll.js

```diff
@@ -1,6 +1,28 @@
-const sumAll = function() {
+const sumAll = function (rangeStart, rangeEnd) {
+  let sum = 0;
+  if (
```

Comment:
Just this one little space
```suggestion
  let sum = 0;
  
  if (
```